src/frontend/src/flows/manage.ts

```diff
@@ -87,7 +138,7 @@ const pageContent = (userNumber: bigint, devices: DeviceData[]) => html`
           <div class="labelWithAction">
             <label id="deviceLabel">Recovery mechanisms</label>
             <button class="labelAction" id="addRecovery">
```

Comment:
you shouldn't disable the recovery button if there is already two recovery devices?

Reply:
I just noticed an issue with the recovery bit; will address separately